docs/flink/flink-getting-started.md

```diff
@@ -44,14 +44,14 @@ Iceberg integration for Apache Flink is available for Flink versions 1.12, 1.13,
 
 ## Preparation when using Flink SQL Client
 
-To create iceberg table in flink, we recommend to use [Flink SQL Client](https://ci.apache.org/projects/flink/flink-docs-stable/dev/table/sqlClient.html) because it's easier for users to understand the concepts.
+To create iceberg table in flink, we recommend to use [Flink SQL Client](https://nightlies.apache.org/flink/flink-docs-release-1.14/docs/dev/table/sqlclient/) because it's easier for users to understand the concepts.
 
-Step.1 Downloading the flink 1.11.x binary package from the apache flink [download page](https://flink.apache.org/downloads.html). We now use scala 2.12 to archive the apache iceberg-flink-runtime jar, so it's recommended to use flink 1.11 bundled with scala 2.12.
+Step.1 Downloading the flink 1.14.x binary package from the apache flink [download page](https://flink.apache.org/downloads.html). We now use scala 2.12 to archive the apache iceberg-flink-runtime jar, so it's recommended to use flink 1.14 bundled with scala 2.12.
 
 ```bash
-FLINK_VERSION=1.11.1
+FLINK_VERSION=1.14.0
```

Comment:
Nit: Maybe consider making this 1.14.4 (or whatever the latest is)? That's what they'll get if they go to the flink downloads page, but it's a minor detail either way.

Reply:
Edit - I don't think this is necessary given that we refer to 1.14.0 in much of the rest of the document.

Reply:
Yes, i have change it to 1.15.0